src/main/java/uk/gov/hmcts/reform/sendletter/healthcheck/SendLetterHealthIndicator.java

```diff
@@ -2,21 +2,15 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.actuate.health.Health;
 import org.springframework.boot.actuate.health.HealthIndicator;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.stereotype.Component;
 
-@Component
-@ConditionalOnProperty(prefix = "send-letter", name = "url")
```

Comment:
does this prevent disabling the autowiring of the health indicator?

Reply:
As provided info in description: no need for component presence here as indicator is beaned up in autoconfiguration. also duplicately wrapped with `conditional on property` annotation. spring v2.1 has removed bean override by default so application is not loading in case of this matter.

p.s. this is why integration tests prove the point things are working as expected 😉

Reply:
i see, i read the description but didn't quite understand 👍